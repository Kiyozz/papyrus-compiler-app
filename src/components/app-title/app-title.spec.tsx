import { shallow } from 'enzyme'
import React from 'react'
import AppTitle from './app-title'

describe('<AppTitle>', () => {
  it('renders correctly', () => {
    const component = (
      <AppTitle>Toto</AppTitle>
    )

    expect(component).toBeTruthy()
  })

  it('matches snapshot and be a h1', () => {
    const component = shallow((
      <AppTitle>toto</AppTitle>
    ))

    expect(component).toMatchSnapshot()
    expect(component.find('h1')).toHaveLength(1)
  })

  it('matches snapshot and accept a classname', () => {
    const component = shallow((
      <AppTitle className="yo">toto</AppTitle>
    ))

    expect(component).toMatchSnapshot()
    expect(component.find('h1.yo')).toHaveLength(1)
  })
})
