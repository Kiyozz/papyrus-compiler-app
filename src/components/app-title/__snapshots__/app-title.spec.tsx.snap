// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppTitle> matches snapshot and accept a classname 1`] = `
<h1
  className="app-title yo"
>
  toto
</h1>
`;

exports[`<AppTitle> matches snapshot and be a h1 1`] = `
<h1
  className="app-title"
>
  toto
</h1>
`;
